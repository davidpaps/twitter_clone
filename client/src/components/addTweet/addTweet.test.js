import React from "react";
import { shallow } from "enzyme";
import AddTweet from "./addTweet";

describe("Add Tweet", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<AddTweet />)));

  it("should render a <div />", () => {
    expect(wrapper.find("Fragment").length).toEqual(1);
  });

  it("should render a <form />", () => {
    expect(wrapper.find("form").length).toEqual(1);
  });

  it("should render a <button />", () => {
    expect(wrapper.find("button").length).toEqual(1);
  });
});
