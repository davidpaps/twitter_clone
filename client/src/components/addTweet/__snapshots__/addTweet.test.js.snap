// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Tweet should rendercorrectly 1`] = `
<form
  className="d-flex mt-5"
  onSubmit={[Function]}
>
  <input
    className="form-control"
    onChange={[Function]}
    placeholder="Add a Tweet!"
    required={true}
    type="text"
    value=""
  />
  <button
    className="btn btn-outline-success btn-sm"
  >
    Add
  </button>
</form>
`;
